ls1intum/Orion
# [feature] add return button

### Motivation and Context
Resolves #33

### Description
Adds a button on the toolbar that allows to always return to the configured Artemis Base URL. Also switches to a more reliable way of overwriting the user agent.

### Steps for Testing
1. Install the release as described in [the readme](https://github.com/ls1intum/Orion/blob/master/README.md#testing-of-pull-requests)
2. Press the newly created button under various circumstances
3. Make sure after returning the submit button is available
4. Be aware #49 is fixed but not yet deployed

### Screenshots
![button](https://user-images.githubusercontent.com/61357727/118527627-c45b3100-b741-11eb-9672-504083761cee.png)
